esmvaltool/cmorizers/data/downloaders/datasets/cds_satellite_soil_moisture.py

```diff
@@ -0,0 +1,78 @@
+"""Script to download CDS-SATELLITE-SOIL-MOISTURE from the CDS."""
```

Comment:
How does this script differ from `esmvaltool/cmorizers/data/download_scripts/download_cds_satellite_soil_moisture.py`? Do we need to keep both in this PR?

Reply:
NO, this is the adapatation of the other onw. it should be removed